compose/celery/Dockerfile

```diff
@@ -1,9 +1,15 @@
 FROM python
 WORKDIR /app
 
+# RUN apt-get update
+# RUN apt-get install -y software-properties-common
+
+#RUN add-apt-repository ppa:sumo/stable
+RUN apt-get update
+RUN apt-get install -y sumo
```

Comment:
> I am not sure why every container needs sumo... Could you explain that to me, please?

Of course! Celery runs in its own service/conainer where the started processes run too. That means, the content of the method `_run` runs in this service, including [this line](https://github.com/BP2022-AP1/bp2022-ap1/pull/422/files#diff-f429acc14efc274f7ba6f9aa41f241c1c5cfa0e55b1b54e0226359610b12c60cR81), where we check for sumo-binaries. Therefore we have to install Sumo because the simulation will be run in the started celery process.

> please don't comment out the ppa, because then we will get a version mismatch between Traci and sumo (the default debian package index is outdated)

I tried 😇 Maybe you can help me out here. I couldn't add the repository, because of the following error: `repository/PPA does not have a Release file?`. Do you have another idea than building sumo manually?

Reply:
> Okay, but we have like 5 containers now, why does every one of them need sumo

Only the development, production, and celery components install sumo the moment. Because sumo only runs in the celery worker, I can remove it from the development and production component. Thanks for the hint. Can we remove it from the github action script too?

Reply:
> I will look into that

The problem is that the python container is based on `debian-bullseye`, but sumo does not distribute binaries for that version (see [here](https://launchpad.net/~sumo/+archive/ubuntu/stable) ~> *Technical details about this PPA* ~> *Choose your ubuntu version*)). There are at least two options now: 
1. Use an other base image like this (the image does not work because it installs an outdated version of python):
```dockerfile
FROM debian:bullseye
WORKDIR /app

RUN echo "deb https://ppa.launchpadcontent.net/sumo/stable/ubuntu bullseye main" >> /etc/apt/sources.list.d/sumo.list
RUN apt-get update && apt-get install -y sumo python3 python3-pip

RUN pip install poetry

COPY . /app
RUN poetry install
CMD poetry run celery -A make_celery worker --loglevel=info
```
2. build sumo ourselves (will probably be a pain)